### ISSUE TITLE
Hue Light Node - Color Temperature setting

### ISSUE THREAD

**Issue by rue67** [Supergiovane/node-red-contrib-knx-ultimate#260]
Hello Max,

we had recently some conversation in knx-user-forum.de regarding KnxUltimateHueLight-Node.
You wanted me to report here my findings.

------

Feature request regarding color temperature setting:

It would be great to adjust color temp. by wall mount
GTS2 (knx) in parallel with node-red dashboard slider on mobile device.
But the problem is, that one can only choose between DPT3.007 
(rel. dimming) OR DPT 5.001.

It would be pretty fine if there are two inputs (=Group addresses) 
for this, DPT3.007 AND DPT5.001 each (as already done for brightness setting!)

--------

My testbench:

Platform Rasperry Pi Model 3B
with headless Debian bullseye (ver. 11)
Kernel 6.1

Node-Red:
3.0.2
node-red-contrib-knx-ultimate
2.1.34 / 2.1.39
node-red-dashboard
3.5.0
node-red-node-pi-gpio
2.0.6

Philips Hue-Bridge
(square version)

Philips Hue white ambiance lamp 8.5W
(tuneable white)

**Comment by rue67** [comment]
Works, thank you!
